src/providers/v1/V1ProjectMetadataProvider.tsx

```diff
@@ -26,6 +26,11 @@ export function V1ProjectMetadataProvider({
   return (
     <ProjectMetadataContext.Provider
       value={{
+        refetchProjectMetadata: () => {
+          throw new Error(
```

Comment:
Since this is not used anywhere yet, figured no point in making it 'work'

Reply:
since there's a default value defined in contexts/projectMetadataContext.ts doing the same thing, isn't this unnecessary?

Reply:
The value for `metadata` is passed in by getStaticProps. We could work it to make it use either or, but I think it is out of scope of this PR and only will really enhance v1 projects